app/templates/admin/sales.hbs

```diff
@@ -20,7 +20,7 @@
         {{#link-to 'admin.sales.fees' class='item'}}
           {{t 'Fees'}}
         {{/link-to}}
-        {{#link-to 'admin.sales.status' class='item'}}
+        {{#link-to 'admin.sales.invoice' class='item'}}
```

Comment:
shouldn't it be `invoices` too? I guess the route name and file names must be same.

Reply:
@Anupam-dagar you are right. pushing the fix.